src/elm/Page/Profile/Claims.elm

```diff
@@ -0,0 +1,347 @@
+                                }
+
+                    else
+                        UR.init newModel
+                            |> UR.addExt (Just (VoteClaim claimId vote) |> LoggedIn.RequiredAuthentication)
+
+                _ ->
+                    model
+                        |> UR.init
+
+        GotVoteResult claimId (Ok _) ->
+            case model.status of
+                Loaded profile ->
+                    let
+                        maybeClaim : Maybe Claim.Model
+                        maybeClaim =
+                            List.find (\c -> c.id == claimId) profile.claims
+
+                        message val =
+                            [ ( "value", val ) ]
+                                |> loggedIn.shared.translators.tr "claim.reward"
+                    in
+                    case maybeClaim of
+                        Just claim ->
+                            let
+                                value =
+                                    String.fromFloat claim.action.verifierReward
+                                        ++ " "
+                                        ++ Eos.symbolToSymbolCodeString claim.action.objective.community.symbol
+                            in
+                            { model
+                                | status = Loaded profile
+                            }
+                                |> UR.init
+                                |> UR.addExt (LoggedIn.ShowFeedback LoggedIn.Success (message value))
+                                |> UR.addCmd (Route.replaceUrl loggedIn.shared.navKey (Route.ProfileClaims model.accountString))
+
+                        Nothing ->
+                            model
+                                |> UR.init
+
+                _ ->
+                    model |> UR.init
+
+        GotVoteResult _ (Err eosErrorString) ->
+            let
+                errorMessage =
+                    EosError.parseClaimError loggedIn.shared.translators eosErrorString
+            in
+            case model.status of
+                Loaded claims ->
+                    { model
+                        | status = Loaded claims
+                        , claimModalStatus = Claim.Closed
+                    }
+                        |> UR.init
+                        |> UR.addExt (ShowFeedback LoggedIn.Failure errorMessage)
+
+                _ ->
+                    model |> UR.init
+
+
+profileClaimQuery : LoggedIn.Model -> String -> Cmd Msg
+profileClaimQuery ({ shared } as loggedIn) accountName =
+    Api.Graphql.query
+        shared
+        (Cambiatus.Query.profile { account = accountName } (selectionSet loggedIn.selectedCommunity))
+        ClaimsLoaded
+
+
+selectionSet : Eos.Symbol -> SelectionSet ProfileClaims Cambiatus.Object.Profile
+selectionSet communityId =
+    SelectionSet.succeed ProfileClaims
+        |> with (Profile.claims (\l -> { communityId = Present (Eos.symbolToString communityId) }) Claim.selectionSet)
```

Comment:
@amiskov this adds a filter to only show profile claims of the selected community